package-res/resources/web/pentaho/visual/role/mapping.js

```diff
@@ -513,21 +513,23 @@ define([
 
         props: [
           /**
-           * Gets or sets the fixed measurement level in which the visual role is to operate on.
+           * Gets or sets the fixed measurement level in which the associated visual role is to operate on.
```

Comment:
Suggest:

 * Gets or sets the fixed measurement level on which the associated visual role is to operate.